python/lsst/pipe/tasks/dcrAssembleCoadd.py

```diff
@@ -112,7 +112,7 @@ class DcrAssembleCoaddConfig(CompareWarpAssembleCoaddConfig):
         dtype=float,
         doc="Maximum relative change of the model allowed between subfilters."
             "Set to zero to disable.",
-        default=4.,
+        default=10.,
```

Comment:
Is this to compensate for the rescaling? It seems a bit arbitrary.

Reply:
No, it's unrelated. I found the results improved very slightly with this change while I was testing frequency regularization. I can revert it if you think that would be cleaner.